tests/unit/conftest.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import pytest
+from ops.testing import Harness
+
+from charm import HydraCharm
+
+
+@pytest.fixture()
+def harness():
+    harness = Harness(HydraCharm)
+    harness.set_model_name("hydra-model")
```

Comment:
Could you change it to avoid conflicts with another opened pr?
```suggestion
    harness.set_model_name("testing")
    harness.set_leader(True)
```